src/VisualStudio/TestUtilities/Window/EditorWindow.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities.Remoting;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    public class EditorWindow
```

Comment:
We should call this ActiveEditorWindow to make it clear it's bound to the active editor. I suspect we'll want to refactor that eventually since we might have some tests that have two windows and jump between them...?